src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -194,6 +194,16 @@ private static String getDwpIssuingOffice(SyaCaseWrapper syaCaseWrapper) {
         String result = null;
         switch (Benefit.getBenefitByCode(benefitType)) {
             case UC:
+                if (mrnIsNotProvided(syaCaseWrapper) || (!mrnIsNotProvided(syaCaseWrapper) && StringUtils.isBlank(syaCaseWrapper.getMrn().getDwpIssuingOffice()))) {
```

Comment:
Can we just change this switch statement to an if/else statement so we don't have to copy and paste code?

Reply:
🤯

Reply:
Looks a lot better now after refactor